doc/classes/Engine.xml

```diff
@@ -67,12 +71,14 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether or not the editor is running.
```

Comment:
Per writing guidelines, boolean methods/members prefer "[code]true[/code]" over "whether or not".

Returns [code]true[/code] if the editor is running.